js/main.js

```diff
@@ -412,7 +414,7 @@ function persistConfig(config) {
   let json = JSON.stringify(config.charts);
   json = json.replace(/\\"/g,'\\\\"');
   return $.ajax({
-    url: `lib/persist.php?pid=${config.pid}`,
+    url: `${endpointUrls['lib/persist.php']}`,
```

Comment:
One thing that isn't clear to me is how the project ID gets into the request now. Is it being added when the url is constructed by `getUrl`?

Reply:
I believe so. I'm not sure at which point it's happening. In a previous version of external modules it was in the `getUrl` function.